ci-operator/step-registry/ipi/conf/openstack/techpreview/ipi-conf-openstack-techpreview-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: ipi-conf-openstack-techpreview
+  steps:
+  - ref: openstack-conf-clouds
+  - ref: openstack-conf-resourcenames
+  - ref: openstack-conf-createfips
+  - ref: openstack-conf-generateconfig
+  - ref: openstack-conf-creatednsrecords
+  - ref: openstack-conf-exportcindercredentials
+  - ref: ipi-conf-etcd-on-ramfs
+  - ref: ipi-conf-openstack-precheck
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-conf-techpreview
+  documentation: |-
+    The IPI configure step chain generates the install-config.yaml file based on the cluster profile and optional input files.
```

Comment:
This step documentation doesn't seem to be accurate as to the purpose of this chain